integration-tests/src/checks/invalid_block_hash.rs

```diff
@@ -0,0 +1,48 @@
+use std::env;
+
+use crate::utils::RadioRuntimeConfig;
+use colored::Colorize;
+use poi_radio::{LocalAttestationsMap, MessagesArc, RemoteAttestationsMap};
+use tracing::{debug, error};
+
+use crate::setup::test_radio::run_test_radio;
+
+fn post_comparison_handler(_messages: MessagesArc, _block: u64, _subgraph: &str, _prev_len: usize) {
+}
+
+fn test_attestation_handler(
+    _block: u64,
+    _remote: &RemoteAttestationsMap,
+    _local: &LocalAttestationsMap,
+) {
+}
+
+fn success_handler(messages: MessagesArc) {
+    let messages = messages.lock().unwrap();
+    debug!("{:?}", messages);
+
+    if messages.len() > 0 {
+        error!("{}", "invalid_block_hash test failed".red());
+        std::process::exit(1);
+    }
```

Comment:
do we want a success log printed and exit with code 0?

Reply:
some of the validation one (I think it was this one, invalid_nonce and invalid_sender) are specifically checking for messages to **not** be in the store after a certain interval of listening (currently 1m), so really these 3 tests are treated as successful if they time out, not if they exit with 0